lib/experimental/navigation-overlay.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:group {"lock":{"move":true,"remove":true},"metadata":{"name":"Overlay"},"style":{"dimensions":{"minHeight":"100vh"},"spacing":{"padding":{"top":"var:preset|spacing|30","bottom":"var:preset|spacing|30"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group" style="min-height:100vh;padding-top:var(--wp--preset--spacing--30);padding-bottom:var(--wp--preset--spacing--30)"><!-- wp:group {"metadata":{"name":"Close"},"align":"wide","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"right"}} -->
```

Comment:
the preset value will change from theme to theme. We could replace it with a calculated value, something like: clamp(1rem, calc( var(--wp--style--root--padding-top) + var(--wp-admin--admin-bar--height)), 20rem). This would give you the 1rem/20rem fallback, taking into account the admin bar and the global padding. To make it work inside the markup we need to encode it, and using this kind of value will make the user not see the exact value in the controls but will still allow them to change it.

Reply:
[This](https://github.com/WordPress/gutenberg/blob/19f944db51b5c2f4ef06ba60377e1cb9e3201abd/packages/block-library/src/navigation/style.scss#L514) is what we are trying to emulate, plus adding the admin bar into the mix